app/controllers/searches_controller.rb

```diff
@@ -1,8 +1,10 @@
 class SearchesController < ApplicationController
   before_action :set_page, only: :show
+  LAST_PAGE = 333
```

Comment:
The reason for this number is a little hard to understand just looking at the code. Maybe a comment or link to this PR would be helpful for future contributors.